### ISSUE TITLE
Dynamic add/delete slides

### ISSUE THREAD

**Issue by jessehendriks** [shhdgit/vue-easy-slider#56]
Hello,

I'm using this package to show images. I get the images from an API and check regularly if images are updated (added or deleted). When an image is added it add an extra slide, that works fine. But when a slide is removed, I will get an empty slide (the image is deleted).

Can you help me fix this?

This is the code that I'm using, where screenItems is data from the API
```
<slider 
  animation="fade" 
  height="900px" 
  :indicators="false" 
  :interval="10000" 
  :control-btn="false">
    
    <slider-item
      v-for="item in screenItems"
      :key="item.id">
        <img 
          style="width:100%;height:100%;"
          :src="item.value"
          class="narrowcasting-screen-item">
    </slider-item>

</slider>
```

Hope I'm clear, if you need more information let me know!

**Comment by shhdgit** [comment]
I'll check it asap

**Comment by shhdgit** [comment]
fixed in v5.3.2

**Comment by jessehendriks** [comment]
Thanks!
